creating_packages/package_dev_flow.rst

```diff
@@ -148,8 +148,19 @@ This parameters takes the same parameters as ``package()`` but in this case ther
 | ``install-folder`` |                    |
 |                    |                    |
 | ``build-folder``   |                    |
+|                    |                    |
+| ``package-folder`` |                    |
 +--------------------+--------------------+
 
+There are 2 modes of operation:
+
+- Using ``source-folder`` and ``build-folder`` does not require the execution of the above ``$ conan package``.
+  It will use the ``package()`` method to extract the artifacts from those folders and create the package,
+  directly in the conan local cache.
+- Using the ``package-folder`` argument (incompatible with the above 2), will not use the ``package()`` method,
```

Comment:
Not sure we should state that here there is a way to "split" the flow. I see this as a fix for ``cmake.install()`` and maybe we could add this information at the bottom of this command or just as a note for those users.

Reply:
Is not only a fix for ``cmake.install()``, but also for pre-compiled binaries that already come in the final package layout, and still want to maintain the ``package()`` method to be able to package from build folder. Actually, this is the complete workflow that uses the output of the ``$ conan package`` command.